docs/_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```

Comment:
Pdoc will ignore this theme so it's unnecessary

Reply:
It is required however for GitHub pages to show the page.